zeebe-cluster/templates/statefulset.yaml

```diff
@@ -80,13 +80,14 @@ spec:
           subPath: startup.sh
         - name: data
           mountPath: /usr/local/zeebe/data
+        securityContext:
+          {{ toYaml .Values.podSecurityContext | indent 12 | trim }}
       volumes:
       - name: config
         configMap:
           name: {{ tpl .Values.global.zeebe . | quote }}
           defaultMode: 0744
       securityContext:
```

Comment:
We should remove this then right?

Reply:
Yes, I forgot :sweat_smile:  - right now it just does nothing, but I'll remove it.